test/common/api_utils.py

```diff
@@ -135,12 +143,8 @@ def api_node_select(client,
                 break
 
     # we now have a node ID list
-    if verbosity >= 3:
-        print "Node ID List:"
-        if node_id_list:
-            print node_id_list, '\n'
-        else:
-            print '**** Empty node ID list.\n'
+    if not  node_id_list:
```

Comment:
Because these functions are part of the infrastructure, I considered this clutter and removed it.